lib/strava/api/ratelimit.rb

```diff
@@ -2,13 +2,36 @@
 
 module Strava
   module Api
+    class RatelimitError < StandardError
+      attr_reader :ratelimit
+
+      def initialize(response)
+        super
+        @ratelimit = Strava::Api::Ratelimit.new(response)
+      end
+    end
+
     class Ratelimit
       def initialize(response)
         @response = response
         @headers = response.headers
         @body = response.body
       end
 
+      def exceeded
+        return false unless limit?
+
+        @exceeded ||= if fifteen_minutes_remaining && fifteen_minutes_remaining <= 0
+                        { fifteen_minutes_remaining: fifteen_minutes_remaining }
+                      elsif  total_day_remaining && total_day_remaining <= 0
```

Comment:
Is that a double space after `elsif`? :)

Reply:
🙈